telematic_system/scripts/log_analysis/README.md

```diff
@@ -0,0 +1,119 @@
+# Prerequisite
+- Preferred operating system ubuntu 20 or above
+- Python environment setup
+    1. Install python
+        ```
+        sudo apt update 
+        sudo apt install python3
+        ```
+    2. Check python version
+        ```
+        python3 --version
+        ```
+        Recommended version is `3.10`
+    3. Create a virtual environment. Navigate to `cda-telematics/telematic_system/scripts/log_analysis` directory, and run below command:
+        ```
+        python3 -m venv .venv
+        ```
+    4. Activate virtual environment.
+        ```
+        .venv\bin\activate
+        ```
+        Note: Need to run this command to activate virtual environment every time openning a new terminal.
+- Install depedencies:
+    - Install debian packages 
+        ```
+        sudo apt install libcairo2-dev libxt-dev libgirepository1.0-dev
+
+        ```
+    - Install python packages
+        ```
+        pip install -r requirements.txt
+        ```
+- Clone repos:
+    - Clone cda-telematics GitHub repos
+    ```
+    git clone https://github.com/usdot-fhwa-stol/cda-telematics.git
+    ```
+
+
+# Process V2xHub bridge log 
```

Comment:
Can we also give the expected output from processing this log? Something like generates a CSV file with the Telematics streaming data from the logs formatted into  columns for "Unit Id,Topic ,Payload Timestamp, 
Metadata"

Same for the scripts mentioned below.